mindsdb/libs/phases/data_analyzer/data_analyzer.py

```diff
@@ -175,13 +176,51 @@ def compute_entropy_biased_buckets(hist_y, hist_x):
     return S, biased_buckets
 
 
+def compute_outlier_buckets(outlier_values,
```

Comment:
Hard to compare side by side, but I assume this is just the old logic made into a function?

Or were there any changes made ?

Reply:
Yes, old logic made into a function